lib/bootstrap/environment.rb

```diff
@@ -89,7 +86,7 @@ def pattern_path(path)
   LogStash::Bundler.setup!(bundler_options)
   require_relative "patches/jar_dependencies"
 
-  require ARGV.shift
+  require 'logstash/runner'
```

Comment:
previously we would `require 'logstash-core/lib/logstash/runner.rb''` 
... which if there was another `require 'logstash/runner'` could be problematic